runtimes/nodejs/object-delete.js

```diff
@@ -11,9 +11,25 @@
  *   }
  */
 var CloudObjectStorage = require('ibm-cos-sdk');
-function main(args) {
-  let { cos, params } = getParamsCOS(args, CloudObjectStorage);
-  return cos.deleteObject(params).promise();
+
+async function main(args) {
+  const { cos, params } = getParamsCOS(args, CloudObjectStorage);
+
+  let response;
+  try {
+    response = await cos.deleteObject(params).promise();
+  } catch (err) {
+    return Promise.reject({
```

Comment:
From the doc I read, you should just be able to throw the object here and it will automatically get wrapped in a `Promise.reject`.

Reply:
Updated based on slack convo.